docker/trino/jvm.config

```diff
@@ -13,4 +13,5 @@
 -Djdk.attach.allowAttachSelf=true
 -Djdk.nio.maxCachedBufferSize=2000000
 -XX:+UnlockDiagnosticVMOptions
+-XX:G1NumCollectionsKeepPinned=10000000
```

Comment:
The trino server spits out a line saying this setting is necessary when it's not set (when the JVM is below a certain version).